specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2019-03-01-preview/costmanagement.json

```diff
@@ -1652,10 +1652,10 @@
 				}
 			}
 		},
-		"/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/providers/Microsoft.CostManagement/Showback": {
+		"/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/providers/Microsoft.CostManagement/ShowbackRules": {
```

Comment:
minor comment: resource type names should be lowerCamelCase, i.e. .`showbackRules`

Reply:
ok, i changed it

Reply:
done